src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -108,6 +111,13 @@ public Controls(Subsystems s) {
 			driveController
 					.leftBumper()
 					.whileTrue(new FullTargetCommand(s.launcherSubsystem, s.drivebaseSubsystem, this));
+			driveController
+					.leftBumper()
+					.onTrue(
+							new FullTargetCommand(s.launcherSubsystem, s.drivebaseSubsystem, this)
+									.until(AutoLogic.isReadyToLaunch())
+									.andThen(new WaitCommand(AutoLogic.FEEDER_DELAY))
+									.andThen(new FeederInCommand(s.intakeSubsystem).until(AutoLogic.untilNoNote())));
```

Comment:
Why was this change made? (Note that when the left bumper is pressed, the above binding will be run, scheduling the `FullTargetCommand`, and then this binding will be run, scheduling the `FullTargetCommand.until(...).andThen(...)` composition and canceling the `FullTargetCommand`. ([`Trigger`s](https://github.com/wpilibsuite/allwpilib/blob/main/wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/button/Trigger.java) are polled using [`EventLoop`s](https://github.com/wpilibsuite/allwpilib/blob/main/wpilibj/src/main/java/edu/wpi/first/wpilibj/event/EventLoop.java), which use a [`LinkedHashSet`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/LinkedHashSet.html), which guarantees iteration order matches insertion order, or the order the bindings were made))

Reply:
this was used for debugging auto, ill uncomment it until we need to debug that again